src/devices/tuya.ts

```diff
@@ -777,6 +793,30 @@ const fzLocal = {
             return tuya.fz.datapoints.convert(model, msg, publish, options, meta);
         },
     } satisfies Fz.Converter,
+    // biome-ignore lint/style/useNamingConvention: ignored using `--suppress`
+    backlight_non_relay_pos: {
```

Comment:
```suggestion
    backlightNonRelayPos: {
```

and remove `// biome-ignore` (same for the one above)